packages/bbui/src/Form/Core/Select.svelte

```diff
@@ -102,7 +102,9 @@
   {onOptionMouseenter}
   {onOptionMouseleave}
   isPlaceholder={value == null || value === ""}
-  placeholderOption={placeholder === false ? null : placeholder}
+  placeholderOption={placeholder === false
```

Comment:
NAB: Would be nice if placeholder was of type `string` instead of `string | false`

Reply:
The inclusion of `false` is just for scenarios where we explicitly want to switch off the placeholder behaviour entirely e.g. an enum. If the placeholder value is nullish, svelte will default to "Choose an option" and the user will be able to unintentionally clear the picker. 

This fix still maintains that behaviour, but in the event that it isn't explicitly turned off (`false`), it still needs a final check to ensure that even when its cleared to an empty string `""`, it must switch to the default string or the placeholder option will not be added.